src/app/infobox.ts

```diff
@@ -0,0 +1,8 @@
+export interface infoboxcontent {
+  head1: string;
+  head4: string;
+  para: string;
+  imgurl: string;
+
```

Comment:
Remove extra spaces, and make sure that interface attributes have meaningful names